embrace-android-sdk/src/test/java/io/embrace/android/embracesdk/internal/spans/CurrentSessionSpanImplTests.kt

```diff
@@ -156,6 +159,49 @@ internal class CurrentSessionSpanImplTests {
         }
     }
 
+    @Test
+    fun `crashing results in the session span and active spans being terminated`() {
```

Comment:
Future work, but is this something that is feasible to cover with an integration test that verifies the payload contains a snapshot of active spans? It should be possible to simulate a crash by getting the uncaught exception handler & manually sending an exception